clang/include/clang/APINotes/Types.h

```diff
@@ -444,6 +454,16 @@ class ParamInfo : public VariableInfo {
     NoEscape = Value.value_or(false);
   }
 
+  std::optional<bool> isLifetimebound() const {
+    if (!LifetimeboundSpecified)
+      return std::nullopt;
+    return Lifetimebound;
```

Comment:
```suggestion
    return LifetimeboundSpecified ? Lifetimebound : std::nullopt;
```

Reply:
Ugh, sorry. I forgot about this comment. I will incorporate this into a follow up PR. Thanks a lot for the review!